.changeset/old-oranges-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": patch
+---
+
+The placeholder in the `ClearInputAdornment` is no longer overlapped by a non-visible clear-button
```

Comment:
The issue is caused by ClearInputAdornment, but it has no placeholder. So I'd suggest:
```suggestion
Prevent overlapping placeholders by a non-visible clear-button
```